spec/forms/steps/capital/other_property_type_form_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe Steps::Capital::OtherPropertyTypeForm do
+  subject(:form) { described_class.new(crime_application:) }
+
+  let(:crime_application) { instance_double(CrimeApplication, properties:) }
+  let(:properties) { class_double(Property, where: existing_properties) }
+  let(:existing_properties) { [instance_double(Property, complete?: false)] }
+  let(:property_type) { PropertyType::RESIDENTIAL.to_s }
+  let(:new_property) { instance_double(Property) }
+
+  describe '#save' do
+    before do
+      allow(properties).to receive(:create).with(property_type:).and_return new_property
+
+      form.property_type = property_type
+      form.save
+    end
+
+    context 'when a property of the type exists' do
```

Comment:
Would it be possible to replicate the functionality of this final context block in investment type:
If the user tries to add another property that is of the same property type as one they have already added, but not yet completed then load that one up. Only create a new one once the existing one of the same type has been completed